pontoon/base/static/js/l20n_interface.js

```diff
@@ -0,0 +1,313 @@
+/* Public functions used across different files */
+var Pontoon = (function (my) {
+  return $.extend(true, my, {
+    l20n: {
+
+      /*
+       * Populate FTL translation area with existing data
+       */
+      renderEditor: function (translation) {
```

Comment:
I remember that We already spoke about that issue but I think this will relatively faster and easier to do than to refactor all of the code to the react/angular2.
Could we use a simple template system instead of manual string concatenation? like e.g.
https://github.com/BorisMoore/jquery-tmpl

I think that would be a lot safer and smarter in the face of our previous security issues.